conf/nanopore.config

```diff
@@ -18,6 +18,9 @@ params {
     // Output cram instead of bam files
     outCram = false
 
+    // Experimental - check if the Ns in the fail VCF file have enough evidence to support a reference basecall
+    correctN = false
```

Comment:
change to default to true. It will be a new folder anyway so users can see both the original and the correted

Reply:
done